app/components/carousel/index.ts

```diff
@@ -0,0 +1 @@
+export { Carousel } from './carousel'
```

Comment:
Thank you for using index file :) I you add a component app/components directory, could you also add to storybook? Then everyone can see how the component looks like and how to use it 👍  The icons is same!